staging/src/k8s.io/sample-apiserver/artifacts/example/deployment.yaml

```diff
@@ -26,4 +26,4 @@ spec:
         imagePullPolicy: Never
         args: [ "--etcd-servers=http://localhost:2379" ]
       - name: etcd
-        image: quay.io/coreos/etcd:v3.5.0-rc.0
+        image: quay.io/coreos/etcd:v3.5.0
```

Comment:
this is not new, but is there a reason we're not using the k8s.gcr.io image here?

Reply:
similar to #91257. We'd better move to `k8s.gcr.io`.